java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -213,4 +224,90 @@ public interface BitmapBaseCommands {
      */
     CompletableFuture<Long> bitpos(
             String key, long bit, long start, long end, BitmapIndexType offsetType);
+
+    /**
+     * Reads or modifies the array of bits representing the string that is held at <code>key</code>
+     * based on the specified <code>subCommands</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitfield/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param subCommands The <code>GET</code> subCommands to be performed on the binary value of the
+     *     string at <code>
+     *     key</code>.<br>
+     *     <ul>
+     *       <li>{@link BitFieldGet} gets the value in {@link Offset} or {@link OffsetMultiplier}
+     *           based on encoding being {@link SignedEncoding} or {@link UnsignedEncoding}.
+     *       <li>{@link BitFieldSet} sets the value in {@link Offset} or {@link OffsetMultiplier}
+     *           based on encoding being {@link SignedEncoding} or {@link UnsignedEncoding}.
+     *       <li>{@link BitFieldIncrby} increases or decreases the value in {@link Offset} or {@link
+     *           OffsetMultiplier} based on encoding being {@link SignedEncoding} or {@link
+     *           UnsignedEncoding}.
+     *       <li>{@link BitFieldOverflow} determines behaviour of {@link BitFieldSet} or {@link
+     *           BitFieldIncrby} when these operations result in under or overflows.
+     *     </ul>
+     *     <br>
+     *     Note:<br>
+     *     <ul>
+     *       <li>{@link Offset} and {@link OffsetMultiplier} must be greater than or equal to 0.
+     *       <li>{@link SignedEncoding} must be less than 64.
+     *       <li>{@link UnsignedEncoding} must be less than 65.
+     *     </ul>
+     *
+     * @return An <code>array</code> of results from subcommands.
+     *     <ul>
+     *       <li>{@link BitFieldGet} returns the value in {@link Offset} or {@link OffsetMultiplier}.
+     *       <li>{@link BitFieldSet} returns the old value in {@link Offset} or {@link
+     *           OffsetMultiplier}.
+     *       <li>{@link BitFieldIncrby} returns the new value in {@link Offset} or {@link
+     *           OffsetMultiplier}.
+     *     </ul>
+     *
```

Comment:
```suggestion
```
Same.
Please double check that you tell what commands return in their docs.

Reply:
BTW, overflow was missing there - does it return anything?
If ^ not - could be there a situation when user submits 10 commands (including overflow), but receives 7 responses?
If ^ yes - add a test for that and describe this situation in docs

Reply:
Missed that, thanks! No return for `OVERFLOW`.